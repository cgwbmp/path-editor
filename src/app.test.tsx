import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './app';

describe('<App />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('DOM structure not changed', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
