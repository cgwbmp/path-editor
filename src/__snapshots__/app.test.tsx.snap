// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> DOM structure not changed 1`] = `
<div
  className="layout"
>
  <aside
    className="layout--controller"
  >
    <form
      className="form"
      name="point"
      onSubmit={[Function]}
    >
      <fieldset>
        <legend>
          Добавить точку маршрута
        </legend>
        <div
          className="form--group"
        >
          <input
            area-label="Введите название"
            name="title"
            onChange={[Function]}
            placeholder="Введите название"
            value=""
          />
          <button
            name="create"
            title="Добавить"
            type="submit"
          >
            ↵
          </button>
        </div>
      </fieldset>
    </form>
    <ul
      className="list"
    />
  </aside>
  <section
    className="layout--map"
  />
</div>
`;
